github/libp2p.yml

```diff
@@ -286,11 +286,14 @@ repositories:
         - tbenbow
     default_branch: main
     delete_branch_on_merge: false
+    files:
+      .github/workflows/stale.yml:
+        content: .github/workflows/stale.yml
```

Comment:
This is already included further down below as an inline file. Why here again?

Reply:
It's for a different repo. Resolved here: https://github.com/libp2p/github-mgmt/pull/180